lib/manager/composer/utils.ts

```diff
@@ -66,7 +66,7 @@ export function extractContraints(
   composerJson: ComposerConfig,
   lockParsed: ComposerLock
 ): Record<string, string> {
-  const res: Record<string, string> = { composer: '1.*' };
+  const res: Record<string, string> = { composer: '^2.0' };
```

Comment:
Is this the same as 2.*?

Reply:
No, if i remember right, star will match lowest and caret highest V2